non_profit/non_profit/doctype/member/member.py

```diff
@@ -360,3 +409,46 @@ def register_member(fullname, email, rzpay_plan_id, subscription_id, pan=None, m
         ))
 
         return member.name
+
+
+def create_membership(details):
+    membership = frappe.new_doc("Membership")
+    membership.member = details.member
+    membership.membership_type = details.membership_type
+    membership.from_date = details.from_date
```

Comment:
The to and from data should be fetched from a current subscritpion period record.